src/app/router/connectorRoutes.tsx

```diff
@@ -149,6 +151,12 @@ const connectorMap: { [key: string]: ConnectorRoute } = {
     title: i18n.t("translation:choose_connector.btcpay.title"),
     logo: btcpay,
   },
+  nwc: {
```

Comment:
We could also now add NWC to the Umbrel page, right?

Reply:
Yeah, good point! :+1: